app/components/MainComponent.vue

```diff
@@ -125,84 +219,23 @@ export default defineNuxtComponent({
     const githubInfo = getDisplayName(config.public)
     const displayName = computed(() => githubInfo);
 
-    const metricsReady = ref(false);
```

Comment:
before the change all those values were `ref`s but now they are defined in data.
I don't see anything in the Organization tab after api returns metrics